frontend/src/components/profile.jsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-useless-escape */
+
+import { Button, IconButton } from '@mui/material';
+import CreateIcon from '@mui/icons-material/Create';
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import React, { useState } from 'react';
+import '../css/profile.css';
+
+export default function Profile() {
+  const [edit, setEdit] = useState(false);
+
+  return (
+    <div className="profilePage">
+      <div id="pictureAndButton">
+        <AccountCircleIcon sx={{ fontSize: 110 }} color="action" />
+        {
+          (edit === true)
+            ? <div />
+            : (
+              <IconButton onClick={() => setEdit(true)}>
+                <CreateIcon sx={{ fontSize: 30 }} />
+              </IconButton>
+            )
+        }
+      </div>
+      <text className="profileText">Name:</text>
+      <div id="nameInputs">
+        <input className={edit === true ? 'editProfileInput' : 'noEditProfileInput'} type="text" placeHolder="Volunteer" readOnly={!edit} />

```

Comment:
Let's use material-UI textfields with the disabled option linked to a boolean value.

`                <TextField
                  autoComplete="given-name"
                  name="firstName"
                  required
                  fullWidth
                  id="firstName"
                  label="First Name"
                  value={firstName}
                  onChange={(e) => { setFirstName(e.target.value); }}
                  onBlur={checkFirst}
                  error={validFirstName}
                  helperText={validFirstName ? 'Please enter your first name' : ''}

                />`